src/test/java/bio/terra/service/profile/GoogleBillingServiceTest.java

```diff
@@ -142,7 +142,7 @@ private GoogleProjectResource buildProjectResource() throws Exception {
     Map<String, List<String>> roleToStewardMap = new HashMap<>();
     roleToStewardMap.put(role, stewardsGroupEmailList);
 
-    ResourceInfo resourceInfo = bufferService.handoutResource();
+    ResourceInfo resourceInfo = bufferService.handoutResource(false);
```

Comment:
What do you think about making this `false` by default?

Reply:
I think it makes sense to always pass the parameter so that we can never accidentally create a project that needs secure monitoring enabled without actually enabling those services. Requiring the arg makes the programmer explicitly say they do or don't need monitoring.